tutorials/eth-api/using-tenderly.md

```diff
@@ -0,0 +1,175 @@
+
+Some of Tenderly's product offerings by category include the following:
+
+ - Development: debugger, gas profiler 
+ - Integration: simulator, fork testing
+ - Monitoring: alerting, war rooms aid kit
+ - Infrastructure: web3 gateway, actions, simulation API
+
+Most services offered by Tenderly are free to use but you'll need to subscribe to a paid plan for advanced features like real-time alerting and war room functionality. Tenderly supports Moonbeam and Moonriver, but does not support Moonbase Alpha at this time. 
+
+In this tutorial, we're going to explore two of Tenderly's most powerful features, the debugger and the simulator. 
+
+## Checking Prerequisites {: #checking-prerequisites } 
+
+To get started, you will need the following:
+
+ - Have a free [Tenderly Account](https://dashboard.tenderly.co/register?utm_source=homepage){target=_blank}. You do not need a paid plan to complete this tutorial. 
+
+## Create a Tenderly Project {: #create-a-tenderly-project }
+
+Although not strictly required, it's a good idea to create a Tenderly project to keep things organized and access more of Tenderly's available features. Underneath the **Select Project** dropdown you can press **Create Project** or head to [this Create Project link](https://dashboard.tenderly.co/projects/create){target=_blank}. 
+
+Give your project a name and then press **Create Project**. Although you can change your project name at a later point, the URL will remain the original one you created. 
+
+![Create a Tenderly account](/images/tutorials/eth-api/using-tenderly/tenderly-1.png)
+
+There is a limit of one project with a free account, however, you can have multiple smart contracts under the purview of a single project.
+
+## Add Smart Contracts {: #add-smart-contracts }
+
+Adding a smart contract to your Tenderly project is akin to bookmarking it. While not required, adding a contract will unlock additional Tenderly features over simply searching for the contract in the Tenderly platform. 
+
+To add a smart contract to your Tenderly project, Click on the **Contracts** tab under the Inspect heading, then Click **Add Contracts**. Then, take the following steps:
+
+1. Enter the address of the contract. For this tutorial, we'll be using the Multichain USDC contract `0x818ec0a7fe18ff94269904fced6ae3dae6d6dc0b` 
+2. Select the network the contract is deployed to. We'll select Moonbeam in this case
+3. Give the contract a name to help you recognize it in the dashboard
+4. Press **Add Contract** 
+
+![Add a smart contract](/images/tutorials/eth-api/using-tenderly/tenderly-2.png)
+
+## Simulate a Transaction {: #simulate-a-transaction }
+
+Simulations allow you to see how a transaction will execute without actually sending it to execute on the blockchain. You can simulate a transaction against any point in time, or simply the latest block. 
+
+Let's craft a transaction to simulate against the Moonbeam network by taking the following steps: 
```

Comment:
this jumps over to simulations while the user is still on the contracts tab, so just add in that they have to switch to the Simulator tab